src/lib/ector/software/Ector_Software.h

```diff
@@ -10,17 +10,17 @@
 #ifdef _WIN32
 # ifdef EFL_BUILD
 #  ifdef DLL_EXPORT
-#   define EAPI __declspec(dllexport)
+#   define ECTOR_API __declspec(dllexport)
 #  else
 #   define EAPI
 #  endif
 # else
-#  define EAPI __declspec(dllimport)
+#  define ECTOR_API __declspec(dllimport)
 # endif
 #else
 # ifdef __GNUC__
 #  if __GNUC__ >= 4
-#   define EAPI __attribute__ ((visibility("default")))
+#   define ECTOR_API __attribute__ ((visibility("default")))
 #  else
 #   define EAPI
 #  endif
```

Comment:
Should it use `EFL_BUILD` as a guard? If not, we could remove this part and include `ector_api.h`

Reply:
I think so too, but just to be sure will wait for another one to see before changing it.